diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -1758,7 +1762,7 @@ export class ContentGame extends React.Component {
             const sent_time = latestMoveSuggestion.time_sent;
             if (
                 this.state.annotatedMessages.hasOwnProperty(sent_time) &&
-                this.state.annotatedMessages[sent_time] === "reject"
+                (this.state.annotatedMessages[sent_time] === "reject" || this.state.annotatedMessages[sent_time] === "reject")
```

Comment:
What does this code do? It looks like it's of the form `foo || foo`.

Reply:
it was supposed to be "replace", which dismisses the current showing advice when user requests a new one. i've pushed a fix